spec/abci++/v0.md

```diff
@@ -0,0 +1,161 @@
+# Tendermint v0 Markdown pseudocode
+
+This translates the latex code for Tendermint consensus from the Tendermint paper into markdown.
+
+### Initialization
+
+```
+h_p = 0
+round_p = 0
+step_p is one of {propose, prevote, precommit}
+decision_p = nil
+lockedValue_p = nil
+validValue_p = nil
+validRound_p = -1
```

Comment:
Hrmm, cc @milosevic @konnov @josef-widder for thoughts on this. 

Its not super clear to me, since I think removing it may make it easier to conclude that when height / round increments, it increments for every validator.

(Its also not entirely clear to me, how / if ever this markdown will get merged)

Reply:
I agree that the subscript `_p` is not optimal. However, it is crucial to see who is executing the code. This prefix is the only indication that the code is executed by a process `p`.

Reply:
Given that some `upon` rules work as pattern matching, it is very important to distinguish the process variables (e.g., `round_p`) from those "temporary" variables (e.g., `vr`, `r`).